packages/core/src/internal/motion/utils.ts

```diff
@@ -347,7 +347,7 @@ export function filterAnimationsByUpdatedProperties(
   animations: PropertyDrivenAnimation,
   updatingProps: Map<string, any>
 ): PropertyDrivenAnimation | null {
-  if (animations === null || animations === undefined || animations === {}) {
+  if (animations === null || animations === undefined) {
```

Comment:
TypeScript did not allow `animations === {}` because that would always result in a true value no matter what `animations` was set to.